detox/ios/Detox/DetoxAppDelegateProxy.m

```diff
@@ -23,6 +23,11 @@ - (UIWindow *)overlayWindow
 	return self;
 }
 
+- (UIViewController *) rootViewController
```

Comment:
Let's try with
```objc
- (BOOL)_canBecomeKeyWindow
{
	return NO;
}
```
instead and see if that helps.

Reply:
It works. Thanks for you suggestion. I will update this PR soon.